tools/kolla-images.py

```diff
@@ -39,15 +39,19 @@
 
 # Maps a Kolla image to a list of containers that use the image.
 IMAGE_TO_CONTAINERS_EXCEPTIONS: Dict[str, List[str]] = {
+    "dnsmasq": [
+        "ironic_dnsmasq",
+    ],
     "haproxy": [
         "glance_tls_proxy",
+        "haproxy",
         "neutron_tls_proxy",
     ],
     "mariadb-server": [
         "mariadb",
         "mariabackup",
     ],
-    "neutron-eswitchd": [
+    "neutron-mlnx-agent": [
         "neutron_mlnx_agent",
```

Comment:
Think we need to add eswitchd to this list. Both neutron-mlnx-agent and neutron-eswitchd use the neutron-mlnx-agent image. Example from customer site:
```
3701af5de4ca   172.25.2.221:80/stackhpc/neutron-mlnx-agent:2023.1-rocky-9-20240809T090211                  "dumb-init --single-…"   7 days ago    Up 7 days                          neutron_eswitchd
a2b5b6ca72c6   172.25.2.221:80/stackhpc/neutron-mlnx-agent:2023.1-rocky-9-20240809T090211                  "dumb-init --single-…"   7 days ago    Up 7 days                          neutron_mlnx_agent
```

Reply:
You are right, hawk eye. This wasn't caught because neutron-mlnx-agent isn't in the list of supported images in pulp.yml, although I did see the issue before I filtered out unsupported images.